imap_processing/cli.py

```diff
@@ -241,67 +243,159 @@ def download_dependencies(self):
             file_list.append(imap_data_access.download(return_query[0]["file_path"]))
         return file_list
 
-    @abstractmethod
+    def upload_products(self, products: list[str]):
+        """
+        Upload data products to the IMAP SDC.
+
+        Attributes
+        ----------
+        products: list[str]
```

Comment:
```suggestion
        products : list[str]
```

Super nit-picky but I think `numpydoc` conventions expect a space on either side of the colon.

Reply:
Thanks for the help with my numpydoc formatting. This is the first time I have used it. I just changed my pycharm setting to numpy docstrings so hopefully that will help.